app/views/mail_shared/panel/_row_content_image.html.haml

```diff
@@ -0,0 +1,17 @@
+- last == true ? last_class = "last" : last_class = ""
```

Comment:
@jessieay I wasnt sure if we would get to a point where we aggregate emails that happen in a short period. If someone uploaded three images/receipts, I imagined we would stack them in the future.

Trying to keep the option open.